blackbelt/dependencies.py

```diff
@@ -275,6 +289,58 @@ def license_checker(project_dir):
     return licenses
 
 
+def get_package_tree(project_dir):
+    """
+    Returns the "tree" of the package and its dependencies in
+    the NPM ls json format.
+    """
+    output = run(['npm', 'ls', '--json'], cwd=project_dir, check=False)
+    return json.loads(output)
+
+
+def get_package_breadcrumbs(package_tree, name, version):
+    """
+    Takes a NPM ls JSON tree and looks up the paths to the given
```

Comment:
Nitpick, npm spells lowercase

Reply:
Ah, never saw lowercased acronyms before.

```
(venv) $ git add -p
diff --git a/blackbelt/dependencies.py b/blackbelt/dependencies.py
index a6c3fb2..f0808c4 100644
--- a/blackbelt/dependencies.py
+++ b/blackbelt/dependencies.py
@@ -300,7 +300,7 @@ def get_package_tree(project_dir):

 def get_package_breadcrumbs(package_tree, name, version):
     """
-    Takes a NPM ls JSON tree and looks up the paths to the given
+    Takes a npm ls JSON tree and looks up the paths to the given
     dependency (name and version).

     Returns an array of paths. Where a path is an array of
Stage this hunk [y,n,q,a,d,e,?]? y

(venv) $ git commit --amend
g[kylef/crumbs b9fbcf8] feat(dep): Display path that a problematic dependency is found
 Date: Tue Sep 4 11:52:59 2018 +0100
 1 file changed, 67 insertions(+), 1 deletion(-)
```

Reply:
They consistently write it lowercase everywhere on their site and in the past I've read something like they transitioned away from treating the word as an acronym. They make fun of it in the top left corner of the https://www.npmjs.com/ site, where there are random npm acronyms generated.